CHANGELOG.md

```diff
@@ -33,9 +33,10 @@ Changelog for NeoFS Node
 - `neofs-cli object head --main-only` no-op flag (#3509)
 - IR's estimations timer management (#3539)
 - `neofs-adm fschain estimations` command (#3572)
+- `--oid` flag from CLI object search commands (#3603)
```

Comment:
what does this refer to? same below

Reply:
Ah, this was sitting for a while in the queue, to be fixed.